### ISSUE TITLE
Update tough-cookie for CVE-2023-26136

### ISSUE THREAD

**Issue by dsuresh-ap** [request/request#3467]
Please update tough-cookie for the [CVE-2023-26136](https://nvd.nist.gov/vuln/detail/CVE-2023-26136)

**Comment by prithwirajbose** [comment]
> Please update tough-cookie for the [CVE-2023-26136](https://nvd.nist.gov/vuln/detail/CVE-2023-26136)

Yes, absolutely need this. overrides doesn't work and practically no clean was to get rid of the tough-cookie vulnerability!

**Comment by dsuresh-ap** [comment]
This library isn't maintained anymore. We switched to axios
